### PR TITLE
 TRW-1557 - added time picker to universal search form

### PR DESCRIPTION
**Pull request by troyblank**

### CODE REVIEW

**Comment by troyblank** [comment]
This adds the time picker to universal search as well as adjusted the date picker to work in the demo.

File: components/timePicker/timePicker.js [hunk 1]
```diff
@@ -29,18 +29,45 @@ export default class Timepicker extends PureComponent {
...
+            this.broadcastChange();
         });
     }
 
     onSelectionClick = (selectionRange) => {
-        this.setState({ selectionRange });
+        this.setState({ selectionRange }, this.broadcastChange);
+    }
+
+    getPettyValue() {
```
Comment (jasonhale) [inline components/timePicker/timePicker.js]: I'm guessing this is supposed to be "getPrettyValue".  If so, don't forget to update it's references.

Comment (troyblank) [reply components/timePicker/timePicker.js]: haha yup! thanks!

Comment (troyblank) [reply components/timePicker/timePicker.js]: @jasonhale  ok fixed!

**Comment by mathom** [comment]
[100% (0.0%) vs master 100%](https://jenkins.topbox.io/job/Topbox/job/rci-components/job/PR-657/2/)

**Pull request merged**